packages/lesswrong/lib/eaGivingSeason.ts

```diff
@@ -7,12 +7,12 @@ export const donationElectionFundraiserLink = "https://www.givingwhatwecan.org/f
 export const setupFundraiserLink = "https://www.givingwhatwecan.org/fundraisers";
 export const postsAboutElectionLink = "/topics/donation-election-2023";
 
-export const votingOpensDate = new Date("2023-12-01");
+export const votingOpensDate = new Date("2023/12/01");
```

Comment:
I was seeing some of the dates be off by one (like below, the first theme week should start on Nov 7). I think this fixes it. (See more [here](https://dygraphs.com/date-formats.html))

<img width="977" alt="Screen Shot 2023-11-05 at 3 06 30 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/5886f24a-651f-4c9f-a6c0-399917f6b183">

Reply:
This seems a bit error prone to me since we'd need to always remember to format dates like this going forward. I think the correct way to fix it is #8174.

Reply:
I feel like both solutions have the issue of "you need to remember to do this going forward"? And I think at least some of the time we do want the date to be based on the local time rather than a fixed time (like, we probably want the banner here to highlight "Effective Giving Spotlight" until midnight on Nov 13 in my local time).

It should be fine to keep both our changes right?

Reply:
Wait, what's this? Using slashes makes it into a date not a datetime???